actors/builtin/miner/policy.go

```diff
@@ -141,6 +144,15 @@ func precommitDeposit(qaSectorPower abi.StoragePower, networkQAPower abi.Storage
 	return InitialPledgeForPower(qaSectorPower, networkQAPower, networkTotalPledge, epochTargetReward, circulatingSupply)
 }
 
+// Determine maximum number of deal miner's sector can hold
+func dealPerSectorLimit(size abi.SectorSize) uint64 {
+	maxDeals := uint64(size / DealLimitDenominator)
+	if maxDeals == 0 {
+		maxDeals = 1
```

Comment:
fyi @whyrusleeping @magik6k we didn't hear back about this, but in case you see this we're setting max deals for 512Mib to 3 and 2kib to 1.